processor/src/test/java/org/mapstruct/ap/test/ignore/ErroneousTargetHasNoWriteAccessorMapper.java

```diff
@@ -32,7 +32,7 @@
     ErroneousTargetHasNoWriteAccessorMapper INSTANCE =
         Mappers.getMapper( ErroneousTargetHasNoWriteAccessorMapper.class );
 
-    @Mapping( target = "hasTallons", ignore = true )
+    @Mapping(target = "hasTallons", constant = "true")
```

Comment:
Me too :smile:. For the record, it should be [talon](https://en.wikipedia.org/wiki/Claw). With one 'l'. Although, that would not get me home-free in the urban dictionary either 😄